services/state/common.proto

```diff
@@ -0,0 +1,60 @@
+syntax = "proto3";
+
+package proto;
+
+/*-
+ * ‌
+ * Hedera Network Services Protobuf
+ * ​
+ * Copyright (C) 2018 - 2023 Hedera Hashgraph, LLC
+ * ​
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * ‍
+ */
+
+option java_package = "com.hederahashgraph.api.proto.java";
+// <<<pbj.java_package = "com.hedera.hapi.node.state.common">>> This comment is special code for setting PBJ Compiler java package
+option java_multiple_files = true;
+
+/**
+ * Message representing a uint256 value.
+ */
+message Int256Value {
+    /**
+     * The number low-order bytes in the 256-bits that contain ones
+     */
+    int32 non_zero_bits = 1;
+    /**
+     * The first 64 bits of a int256 value
+     */
+    int64 first_long = 2;
```

Comment:
I get that `uint64` types don't work well in Java, but having these be `int64` types is awkward too because representing each quarter of the `int256` (and why `int256` and not `uint256`?) with `int64` may lead to each one having random looking signs, and various confusions when writing/reading code (not to mention looking at protobufs themselves). I guess `first_long` is so you know how many of the following named fields to expect - but then why not just represent as `bytes` to be left-padded by user to 32 bytes just like for `SlotKey.value` below?  Could use either left-padding with a field to indicate whether to pad with `0` or `0xff` bytes, or could just say you need to sign-extend it to left.